pkg/controller/appsync/helpers.go

```diff
@@ -54,11 +54,19 @@ func (c *Controller) syncApps(ctx context.Context, apps *clients.AppsResponse) *
 
 		realm.AgencyBackgroundColor = strings.ToLower(app.AgencyColor)
 		realm.AgencyImage = app.AgencyImage
+		realm.DefaultLocale = app.DefaultLocale
 		if err := c.db.SaveRealm(realm, database.System); err != nil {
 			merr = multierror.Append(merr, fmt.Errorf("unable to update agency information: %w", err))
 			continue
 		}
 
+		if res, err := c.db.SyncRealmTranslations(realm.ID, app.Localizations); err != nil {
```

Comment:
Can we add a test for this in helper_test?

Reply:
added tests